chapter_installation/index.md

```diff
@@ -112,10 +123,11 @@ jupyter notebook
 
 At this point, you can open http://localhost:8888 (it usually opens automatically) in your Web browser. Then we can run the code for each section of the book.
 Please always execute `conda activate d2l` to activate the runtime environment
-before running the code of the book or updating MXNet or the `d2l` package.
+before running the code of the book or updating the framework or the `d2l` package.
 To exit the environment, run `conda deactivate`.
 
 
+:begin_tab:`mxnet,pytorch`
```

Comment:
it's not supported yet to have a section in a begin_tab. move this line below `## GPU Support` should fix the issue